src/Content/Item.php

```diff
@@ -223,15 +223,19 @@ public static function replaceTag(string &$body, int $profile_uid, string $tag,
 
 				// select someone by nick in the current network
 				if (!DBA::isResult($contact) && ($network != '')) {
-					$condition = ["`nick` = ? AND `network` = ? AND `uid` = ?",
-						$name, $network, $profile_uid];
+					$condition = [
+						"`nick` = ? AND `network` = ? AND `uid` = ?",
+						$name, $network, $profile_uid
+					];
```

Comment:
Can't this be simplified to:
```suggestion
					$condition = [
						'nick'    => $name,
						'network' => $network,
						'uid'     => $profile_uid,
					];
```
?

Reply:
Done